.github/workflows/windows_build.yml

```diff
@@ -26,14 +25,14 @@ jobs:
           python3 -m pip install -U wheel setuptools pip
           python3 -m pip install -U pywin32 lxml pymavlink numpy matplotlib pyserial opencv-python PyYAML Pygame Pillow wxpython prompt-toolkit scipy
           python3 -m pip install -U openai pyaudio
-          python3 -m pip install -U pyinstaller==6.7.0 packaging 
```

Comment:
WOuld be nice to have comments as to why this pinning is no longer required.  It wasn't added for fun, it fixed a problem (https://github.com/ArduPilot/MAVProxy/pull/1400)